TelCo.ColorCoder/Program.cs

```diff
@@ -1,145 +1,40 @@
 ﻿using System;
 using System.Diagnostics;
 using System.Drawing;
+using TelCo.ColorCoder.Utils;
+using static TelCo.ColorCoder.Utils.Colors;
```

Comment:
I like the code organization. The data is separated into different classes and the functionality is oved to Util class.

To make this distinction clear, you could add a name to using statement like
using Utils = TelCo.ColorCoder.Utils.Colors;

Reply:
Please check now